WcaOnRails/spec/helpers/notifications_helper_spec.rb

```diff
@@ -79,27 +79,28 @@
       end
     end
 
-    context "when signed in as a board member" do
-      let(:board_member) { FactoryBot.create :user, :board_member, :wca_id }
-      let!(:unconfirmed_competition) { FactoryBot.create :competition }
-      let!(:confirmed_competition) { FactoryBot.create(:competition, :confirmed) }
-      let!(:visible_confirmed_competition) { FactoryBot.create(:competition, :confirmed, :visible) }
-      let!(:visible_unconfirmed_competition) { FactoryBot.create :competition, :visible }
-
-      it "shows confirmed, but not visible competitions, as well as unconfirmed, but visible competitions" do
-        notifications = helper.notifications_for_user(board_member)
-        expect(notifications).to eq [
-          {
-            text: "#{confirmed_competition.name} is waiting to be announced",
-            url: admin_edit_competition_path(confirmed_competition),
-          },
-          {
-            text: "#{visible_unconfirmed_competition.name} is visible, but unlocked",
-            url: admin_edit_competition_path(visible_unconfirmed_competition),
-          },
-        ]
-      end
-    end
+    # Make a test for WCAT instead
+    # context "when signed in as a board member" do
+    #   let(:board_member) { FactoryBot.create :user, :board_member, :wca_id }
```

Comment:
Maybe you can keep the same test, but replace `board_member` by [`wcat_member`](https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/spec/factories/users.rb#L79)?

Reply:
Ah, of course. I searched for such a factory but forgot we use the short names :P